cpp/CMakeLists.txt

```diff
@@ -84,3 +87,4 @@ add_subdirectory(cycles_module)
 add_subdirectory(community_detection_module)
 add_subdirectory(pagerank_module)
 add_subdirectory(uuid_module)
+add_cugraph_subdirectory(ccugraph_alg_module)
```

Comment:
Having a custom `add_subdirectory` command is convenient because `CUDA` is not enabled by default.